src/Parser/Stores.php

```diff
@@ -15,22 +15,53 @@ public function parse($data)
         }
 
         foreach ($data as $stores) {
-            $this->results[$stores['storeIdentifier']] = $stores['id'];
+            $this->results[$stores['storeIdentifier']] = [
+                'id' => $stores['id'],
+                'defaultLanguage' => $stores['configuration']['defaultLanguage'],
+                'languageList' => array_map('trim', explode(',', $stores['configuration']['languageList']))
```

Comment:
I could be wrong but I think if you add the space to the explode statement e.g.
```
explode(', ', $stores['configuration']['languageList'])
```
you wouldn't have to trim the strings.

Reply:
It could be a done in your suggested way but I think it is safer to do this way if plentymarkets would change how they separate language in the future :)